sbt-version-policy/src/test/scala/sbtversionpolicy/DependencyCheckReportTest.scala

```diff
@@ -26,14 +26,15 @@ object DependencyCheckReportTest extends BasicTestSuite {
       isCompatible("1.2.3-RC1", "1.2.3-RC1")
     }
     withScheme(VersionCompatibility.EarlySemVer) { (isCompatible, isBreaking) =>
-      isBreaking  ("0.1.1",     "0.1.0")
-      isBreaking  ("0.2.0",     "0.1.0")
-      isBreaking  ("1.0.0",     "0.1.0")
-      isCompatible("1.0.1",     "1.0.0")
-      isBreaking  ("1.1.0",     "1.0.0")
-      isBreaking  ("2.0.0",     "1.0.0")
-      isBreaking  ("1.0.0",     "1.0.0-RC1")
-      isCompatible("1.0.0-RC1", "1.0.0-RC1")
+      isBreaking  ("0.1.1",          "0.1.0")
+      isBreaking  ("0.2.0",          "0.1.0")
+      isBreaking  ("1.0.0",          "1.0.0-RC1")
+      isBreaking  ("1.0.1",          "1.0.0-RC1")
+      isBreaking  ("1.1.0",          "1.0.0-RC1")
+      isBreaking  ("2.0.0",          "1.0.0-RC1")
+      isCompatible("1.0.0-RC1",      "1.0.0-RC1")
+      isCompatible("1.0.1-RC1",      "1.0.0")
+      isCompatible("1.2.1-SNAPSHOT", "1.2.0")
```

Comment:
Do you mind keeping the following existing checks?

~~~ scala
isCompatible("1.0.1", "1.0.0")
isBreaking  ("1.1.0", "1.0.0")
isBreaking  ("2.0.0", "1.0.0")
~~~